packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1645,11 +1648,13 @@ export default class BackupNg {
                       let parentPath
                       if (isDelta) {
                         const vdiDir = dirname(path)
-                        parentPath = (await handler.list(vdiDir, {
-                          filter: filename =>
-                            !isHiddenFile(filename) && isVhd(filename),
-                          prependDir: true,
-                        }))
+                        parentPath = (
+                          await handler.list(vdiDir, {
+                            filter: filename =>
+                              !isHiddenFile(filename) && isVhd(filename),
+                            prependDir: true,
+                          })
+                        )
```

Comment:
Changes related to code formatting, I think we should keep this in order to not doing again next time when we format the code.